content/blog/por-onde-comecar.md

```diff
@@ -0,0 +1,86 @@
+
+Para começar com programação tem que necessariamente sentar a frente de um computador e escrever código. Você pode assistir mil vídeos e ler duzentos livros mas é FUNDAMENTAL que você sente a frente de um computador e escreva programas para aprender. 
+
+É irrelevante se você usa [Windows](https://www.microsoft.com/en-ca/windows), [Linux](https://en.wikipedia.org/wiki/Linux) ou [macOS](https://en.wikipedia.org/wiki/MacOS). Use o que você tem acesso para aprender. Também não se preocupe com qual edidor de programas ou [IDE](https://en.wikipedia.org/wiki/Integrated_development_environment) usar. Escolha algo simples e se concentre em aprender a programar.
+
+Escolha uma linguagem de programação como [Python](https://wiki.python.org.br/DocumentacaoPython) ou [JavaScript](https://www.w3schools.com/js/default.asp) e pronto. Se concentre em aprender uma linguagem apenas pois tentar aprender um monte de coisa ao mesmo tempo só vai gerar frustrações. Fique a vontade para usar outra linguagem caso você não tenha interesse em Python ou JavaScript.
+
+Na seção de [Links do site Osprogramadores](https://osprogramadores.com/links/) você encontrará uma vasta coleção de tutoriais e documentação que irão te ajudar a aprende Python ou JavaScript e outras linguagens de programação. Escolha uma linguagem de programação e siga em frente. Usaremos Python nos exemplos que serão apresentados abaixo.
+
+Vamos começar?
+
+Comece criando um programinha tipo um "Olá Mundo". Imprima uma mensagem e pronto. Veja um exemplo do programa Hello World em Python abaixo.
+
+~~~~
+print("Olá Mundo.")
+~~~~
+
+Uma vez entendido o funcionament do programa Hellow World você parte para algo um pouco maior insere a segunda linha para imprimir a segunda mensagem, conforme exemplo abaixo.
+
+~~~~
+print("Olá Mundo.")
+print("Escreva algo aqui.")
+~~~~
+
+Uma vez isso entendido você vai pesquisar o que é uma variável. Crie um segundo programa e atribua o valor 0 (zero) a uma variável e exiba o valor de A na tela do seu computador.
+
+~~~~
+A=1
+print(A)
+~~~~
+
+Depois pesquise como você pode adicionar 1 ao valor de A e exiba o novo valor de A na tela do seu computador. 
+
+~~~~
+A=0
+print(A)
+A=A+1
+print(A)
+~~~~
+
+Uma vez isso dominado insira uma segunda variável, chamada B, no seu programa e imprima o valor de A + B. 
+
+~~~~
+A=0
+print(A)
+A=A+1
+print(A)
+B=35
+A=A+B
+print(A)
+~~~~
+
+Depois disso entendido parta para algo um pouco mais complicado e entenda como você pode usar um comando tipo "for" para repetir uma sequência de linhas do seu programa várias vezes. Sugestão: Imprima uma mensagem com o seu nome 10 vezes.
+
+Exemplo:
+
+~~~~
+for x in range(10):
+  print("Escreva o seu nome aqui")
+~~~~
+
+Uma vez dominado o uso de um comando de repetição como o "for", pesquise o que é decisão e como funcionam comandos do tipo IF THEN ELSE. Faça um programinha simples usando IF THEN ELSE e procure entender como funciona. 
```

Comment:
s/Uma vez dominado ... "for"/Vamos usar o comando IF para tomar uma decisão... (ou algo assim). Eu acho que tem muita dominação nesse texto. :) :) :)

Reply:
resolvido